content/spin/serverless-ai-tutorial.md

```diff
@@ -232,6 +232,14 @@ Now let's use the Spin SDK to access the model from our app:
 
 {{ startTab "Rust"}}
 
+> Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.
+
+```toml
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
You could ask them to run:

```
$ cargo add serde -F derive
$ cargo add serde_json
```

This might provide a more specific "do this" and save them having to manually edit Cargo.toml?  I don't have strong feelings, just mentioning it as an idea.

Reply:
Thanks added this as an alternative.